config/locales/application_states.yml

```diff
@@ -47,7 +47,7 @@ en:
       unsubmitted: Not sent
       awaiting_provider_decision: Awaiting decision
       inactive: Awaiting decision
-      interviewing: Awaiting decision
+      interviewing: Interviewing
```

Comment:
Should this have a system test?

Reply:
Not sure as this is content in a locale.